File: ESTRUTURAS/2482/main.py
def get_messages(n):
    messages = {}
    for _ in range(n):
        language = input().strip()
        message = input().strip()
        messages[language] = message
    return messages


def formatted_print(m, messages):
    for _ in range(m):
        name = input().strip()
        language = input().strip()
        print(name)
        print(messages[language])
        print()


n = int(input())
messages = get_messages(n)
m = int(input())
formatted_print(m, messages)

File: ESTRUTURAS/2482/main.rb
# frozen_string_literal:true

def get_messages(n)
  messages = {}
  n.times do
    language = gets.strip
    message = gets.strip
    messages[language] = message
  end
  messages
end

def formatted_print(m, messages)
  m.times do
    name = gets.strip
    language = gets.strip
    puts name
    puts messages[language]
    puts
  end
end

n = gets.to_i
messages = get_messages(n)
m = gets.to_i
formatted_print(m, messages)
